nntrainer/include/neuralnet.h

```diff
@@ -207,7 +207,7 @@ class NeuralNetwork {
    * @param[in] X input tensor
    * @retval shared_ptr<const Tensor>
    */
-  sharedConstTensor inference(const Tensor X);
```

Comment:
input for inference must also be `sharedTensors`?
This is to keep the semantics of multiple inputs/multiple outputs.

Reply:
fixed. Thanks.